src/components/TeacherComponents/hooks/downloadAndShareHooks/useResourceFormState.tsx

```diff
@@ -62,19 +62,6 @@ export const useResourceFormState = (props: UseResourceFormStateProps) => {
     useFeatureFlagVariantKey("teacher-download-auth") === "with-login";
   const { isSignedIn, user } = useUser();
 
-  const [hasOnboardingDownloadDetails, setHasOnboardingDownloadDetails] =
-    useState(false);
-
-  useEffect(() => {
-    if (user != null) {
-      // as user has signed in with full onboarding journey on OWA
-      const hasOnboardingDownloadDetails = Boolean(
-        authFlagEnabled && isSignedIn && user.publicMetadata?.owa?.isOnboarded,
```

Comment:
Decided to ignore the feature flag with this incoming change. If a user is signed in and onboarded then we should use the stored details for downloads.